### PR TITLE
fix(env-check): fix windows spawn

### PR DESCRIPTION
**Pull request by cianmSAP**
#724

### CODE REVIEW

File: packages/environment-check/src/checks/getInstalled.ts [hunk 1]
```diff
@@ -33,9 +33,9 @@ async function getExtensionsBAS(): Promise<{ [id: string]: { version: string } }
     const versions = files
         .filter((vsixFile) => isExtensionRequired(vsixFile))
         .reduce((returnObject, current) => {
-            const lastIndex = current.lastIndexOf('-');
-            const id = current.slice(0, lastIndex);
-            const version = current.slice(lastIndex + 1).split('.vsix')[0];
+            const idVersion = current.split('.vsix')[0];
```
Comment (Klaus-Keller) [inline packages/environment-check/src/checks/getInstalled.ts:36]: Hi @cianmSAP, I've noticed that the file name `getInstalled.ts` is camelCase, while all other files are using kebap-case. 
Could you please rename the file to `get-installed.ts`?

**Review by Klaus-Keller — CHANGES_REQUESTED** [review]

File: packages/environment-check/src/checks/getInstalled.ts (hunk 1 above)
Comment (cianmSAP) [reply packages/environment-check/src/checks/getInstalled.ts:36]: Updated

**Review by Klaus-Keller — APPROVED** [review]
- Code looks good
- Built and tested locally on Windows using CLI
- Test coverage looks good

Approved. Thanks @cianmSAP 


While testing I stepped into an issue with `@sap/bas-sdk` which is already reported here: 
https://github.wdf.sap.corp/devx-wing/bas-sdk/issues/95
I've create PR https://github.wdf.sap.corp/devx-wing/bas-sdk/pull/98

**Review by devinea — APPROVED** [review]
Changes look good. 
haven’t tested locally.

**Comment by cianmSAP** [comment]
Thanks @Klaus-Keller @devinea

**Pull request merged**